tests/manual-test-cases/Group5-Functional-Tests/5-29-Remove-Container-VM-OOB.robot

```diff
@@ -0,0 +1,64 @@
+# Copyright 2016-2018 VMware, Inc. All Rights Reserved.
```

Comment:
Is it viable to have this as a _testcase_ in the existing 5-14  out-of-band deletion test? I'm asking because having it as a separate suite means a full nimbus testbed deploy which is a significant timesink.